### ISSUE TITLE
Document or have some better way of getting error codes back from send

### ISSUE THREAD

**Issue by jonespm** [IMSGlobal/caliper-python#143]
Perhaps I'm using this wrong but the only way I could find to get the status code back from the_sensor.send was

- Add `debug=True` to the config caliper.HttpOptions
- Iterate through the client_registry and print out the debug status after calling send
```
for k, client in the_sensor.client_registry.items():
        for debug in client.debug:
                print ("Client {0} last status {1} {2} ".format(client.apiKey, debug.status_code, debug.text))
```

Is there an easier way of getting the last status code back, ideally with debug off? This is the "nice" thing about the requests library is that this is always available. Having this would have saved me a some time as there are many response codes sent by the [endpoint I'm testing](https://rawgit.com/unizin/udp-docs/master/udp/event-data/caliper-endpoint.html#2-responses).

**Comment by ViktorHaag** [comment]
Hmmm, yeah, I was of two minds about that when I wrote what I did... how about this:
- on debug=False, we populate `response` with just the status_code
- on debug=True, we populate `response` with the full requests `response` object?

Does that make sense?

**Comment by ViktorHaag** [comment]
I'm starting to think the distinction between Sensors and Clients in the Python impl was way too cute, and we should have just mapped one Sensor directly to a single transport stack instead of having the Client farm aggregating transports under the Sensor. 8(

**Comment by jonespm** [comment]
Yeah, it seems like all I had was the default client on the sensor. (So I really didn't need this loop) Having a "SimpleSensor" seems like it would work fine. This real value of these libraries seems to generate valid caliper event and entities.

**Comment by ViktorHaag** [comment]
Working on a PR now. 8)

**Comment by ViktorHaag** [comment]
Addressed by PR #144
